packages/docs/site/docs/13-contributing/03-coding-standards.md

```diff
@@ -1,73 +1,52 @@
 
-## Formatting
-
-Formatting is handled automatically by the relevant tools and verified by CI. Relax, sit back, and let the machines do the work.
+We handle code formatting and linting automatically. Relax, type away, and let the machines do the work.
 
 ## Public API
 
-Playground aims to keep the lowest possible API area.
+Playground aims to keep the narrowest possible API scope.
 
-Public APIs are easy to add and hard to remove. It only takes one PR to add a new API, but it may take a thousand to remove it – especially if was already consumed in other projects.
+Public APIs are easy to add and hard to remove. It only takes one PR to introduce a new API, but it may take a thousand to remove it, especially if other projects have already consumed it.
 
-Don't expose anything that is not needed. If you don't need to expose a function, don't. If you don't need to expose a class, don't. If you don't need to expose a constant, don't.
+-   Don't expose unnecessary function, class, constant, or other components.
 
 ## Blueprints
 
-Blueprints are the main way to interact with the WordPress Playground. They are JSON files that describe a set of steps to perform. The Playground will execute these steps in order.
-
-### Batteries
-
-Blueprint steps are meant to be small and focused – like Unix tools. They should do one thing and do it well.
-
-When it comes to building new steps:
-
--   It is best not to build it at all
--   If you must build it, try refactoring an existing step first
--   If you cannot do that, make sure the new step opens up an entirely new feature that doesn't do anything that can be achieved with existing steps
-
-### Composability
-
-Blueprint Steps are meant to be composable. This means that you can run them many times over and still get sensible results.
-
-**Don't:**
-
--   Assume your step won't be run more than once.
--   Assume your step will be run in a specific order.
--   Modify PHP files to define constants.
-
-**Do:**
+Blueprints are the primary way to interact with Playground. These JSON files describe a set of steps that Playground executes in order.
 
--   Add unit tests that run your step multiple times.
--   Store constants in virtual JSON files.
+### Guidelines
 
-### Ease of use
+Blueprint steps should be **concise and focused**. They should do one thing and do it well.
 
-Blueprints are meant to be easy to use and require the minimal amount of effort on the consumer side.
+-   If you need to create a new step, try refactoring an existing one first.
+-   If that's not enough, ensure the new step delivers a new capability. Don't replicate the functionality of existing steps.
 
-**Don't:**
+Blueprint steps should be **idempotent**, so you can run them multiple times and get similar results.
```

Comment:
This isn't the case at the moment, it needs to be considered for Blueprints v2

Reply:
Oh, it's a rewrite of this subsection:
> ### Composability
> Blueprint Steps are meant to be composable. This means that you can run them many times over and still get sensible results.

Which was confusing, because it says _Composability_ but describes _Idempotency_ :-)

What would be an alternative intro (_Blueprint steps should be..._) for these?
> - Assume the step would run more than once.
> - Assume it would run in a specific order.
> - Add unit tests to verify that.
> - Define constants in virtual JSON files—don't modify PHP files.

Reply:
Oh that bit is indeed confusing. Running more than once only refers to having more than one, say, „installPlugin” step :-) but trying to install the same plugin twice would fail — I’d just clarify this bit

Reply:
Coolio. On it